solidity/random-beacon/test/RandomBeacon.GroupCreation.test.ts

```diff
@@ -1048,12 +1092,17 @@ describe("RandomBeacon - Group Creation", () => {
 
             context("with challenge period passed", async () => {
               let tx: ContractTransaction
+              let initialSubmitterBalance: BigNumber
```

Comment:
Not a const? 🤔

Reply:
I think I cannot use const here.
If something is `const` it needs to be initialised on declaration, so I would have to call:
```
 const initialSubmitterBalance = await testToken.balanceOf(
      submitterAddress1
 )
```

If I call it inside `beforeEach`, it won't be visible in `it` testcases.
If I call it in `context`, outside `beforeEach`, it won't work either.